changelog/646.feature.rst

```diff
@@ -0,0 +1 @@
+Add explicit 'logical' choice to --numprocesses flag
```

Comment:
Let's describe this a bit better, and also use proper formatting:

```suggestion
Add ``--numprocesses=logical`` flag, which automatically uses the number of logical CPUs available, instead of physical CPUs with ``auto``.

This is very useful for test suites which are not CPU-bound.
```

Reply:
Updated. 👍